### ISSUE TITLE
508 Error with Combo Box on USAJOBS

### ISSUE THREAD

**Issue by Piglady64** [uswds/uswds-site#1359]
## Title line template: [Title]: Brief description
USAJOBS is receiving a 508 error when using the combo box in the design systemhttps://designsystem.digital.gov/components/combo-box/

## Description
Below is the explanation from the USAJOBS Team Titan Developers: 
The combo box is created via a script in the USWDS 2.0 and because of this, we are unable to address this item on our side.  Someone will need to reach out to mention it to the team that created the design system.  

The fix appears to be to have aria-controls added to the combo box input that's generated.  The validation is that the this input must have both aria-expanded and aria-controls.

The following is the combo box input that has the 508 issue.  There is an aria-expanded property, but not an aria-controls one.
<input role="combobox" type="text" class="usa-combo-box__input" autocomplete="off" autocapitalize="off" aria-expanded="false" aria-describedby="OrganizationID--assistiveHint" aria-autocomplete="list" aria-owns="OrganizationID--list" id="OrganizationID">

## Additional information [optional]
[508 Invite User ATP - Updated.xlsx](https://github.com/uswds/uswds-site/files/7635781/508.Invite.User.ATP.-.Updated.xlsx)
[508 Testing - Module - Invite User · Agency Talent Portal - 11-08.xlsx](https://github.com/uswds/uswds-site/files/7635783/508.Testing.-.Module.-.Invite.User.Agency.Talent.Portal.-.11-08.xlsx)
[Module - Invite User · Agency Talent Portal - Confirmed Violations 13.docx](https://github.com/uswds/uswds-site/files/7635784/Module.-.Invite.User.Agency.Talent.Portal.-.Confirmed.Violations.13.docx)

**Comment by amyleadem** [comment]
@Piglady64 
Hello, I have updated the combobox script to build the input with an aria-controls attribute. Can you review the rendered code in this [preview link](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/al-combobox-508/components/detail/combo-box--default.html) and let me know if this meets expectations? Thanks!

**Comment by Piglady64** [comment]
I had one of my developers look at this and he said it should fix the problem. Once it is implemented we will run another 508 test to make sure and let you know if it has resolved the issue.

Julie Jackson
Agency Talent Portal Product Owner
Office: 202-606-5230
Cell:    202-718-7608
***@***.******@***.***> | www.opm.gov/HRS<http://www.opm.gov/HRS>
[USAJOBS signature block]T

**Comment by mejiaj** [comment]
**[PR available here →](https://github.com/uswds/uswds/pull/4483#pullrequestreview-876516355)**

**Comment by thisisdano** [comment]
Fixed in USWDS `2.13.2`: https://github.com/uswds/uswds/releases/tag/v2.13.2
